build/index.js

```diff
@@ -9,5 +9,13 @@ const template = require.resolve('govuk_template_mustache/views/layouts/govuk_te
 
```

Comment:
There is also an index.js file in the root of this module. Just out of interest what does each one do? I'm unfamiliar with this module so trying to figure out how it works, sorry.

Reply:
The one in build is called by the package.json to generate the template:

  "scripts": {
    "lint": "eslint .",
    "test": "npm run lint",
    "prepublish": "node ./build/index.js"
  },

The top-level index.js takes an app and adds the template to locals. This is used in hof-form-wizard.